libdrf/login/settings.py

```diff
@@ -21,7 +21,7 @@
 
     'JWT_SECRET_KEY': settings.SECRET_KEY,
     'JWT_ALGORITHM': 'HS256',
-    'JWT_VERIFY': True,
+    'JWT_VERIFY': False,
```

Comment:
Is this change really needed? I think this just sets the default value for verifying, and I think it's good if that is True?